spec/integration/basic.spec.js

```diff
@@ -83,6 +83,26 @@ describe( "Basic - Integration", () => {
 				] );
 		} );
 
+		it( "should work with multiple sets", () => {
+			const query = `
+				SELECT *
+				FROM QueryTests
+				ORDER BY id DESC;
+
+				SELECT sum(id)
+				FROM QueryTests`;
+
+			return sql.query( query )
+				.should.eventually.deep.equal( [
+					[
+						{ id: 3, test: "C" },
+						{ id: 2, test: "B" },
+						{ id: 1, test: "A" }
+					],
+					[ { "": 6 } ]
```

Comment:
Totally off-topic, but what would happen here if we had multiple unnamed columns? Could we key these by index maybe?

Reply:
That seems like a better plan.